MAKE/Makefile.vorna_gcc

```diff
@@ -1,9 +1,11 @@
 # Makefile for University of Helsinki's "vorna" Cluster
 #
 # Reccomended modules in .bashrc:
-# module load GCCcore/8.3.0
-# module load OpenMPI/3.1.4-GCC-8.3.0
-# module load Eigen
+# module purge
+# module load GCC/10.2.0
+# module load OpenMPI/4.0.5-GCC-10.2.0
+# Same in one line to past on the command line:
+# module purge ; module load GCC/10.2.0 ; module load OpenMPI/4.0.5-GCC-10.2.0
 
```

Comment:
Add:
remember to compile inside interactive session:
/usr/bin/srun -M vorna --interactive -n 16 --mem=4G --time=00:20:00 -p test --pty bash

Reply:
Done